src/scss/styles.scss

```diff
@@ -0,0 +1,135 @@
+@import "../../node_modules/bootstrap/scss/bootstrap";
+@import "variables";
```

Comment:
This should come first before the `/scss/bootstrap`. You can read more at https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html